### DISCUSSION TITLE
Machine learning capstone

### DISCUSSION THREAD

**Discussion by Stevee136** [teamalgoritma/community#901] [Q&A] [answered]
Project ini disuruh buat laporan dalam R studio sambil menyertakan kode ya kak?

**Comment by adizaen** [comment, marked as answer]
Dear Pak @Stevee136 

Benar sekali Pak. Untuk informasi lengkapnya, Bapak bisa merujuk pada link berikut:
https://rpubs.com/Algoritma/ml-capstone

Semoga menjawab Pak

Best regards,
**Team Algoritma**
